<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="newReaderFileEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="showReaderEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addBookEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="newBookEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="delBookEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="giveBookLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Give book</string>
    </property>
   </widget>
   <widget class="QLabel" name="takeLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Take back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allBooksBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All books</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allReadersBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All Readers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showEditBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>108</y>
      <width>86</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open reader file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBooksButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>86</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addReadersButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>86</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Reader</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
